spec/features/collections/loss_report_spec.rb

```diff
@@ -0,0 +1,40 @@
+
+describe 'Displaying the comparison report in browser' do
+
+  context 'when accessing the comparison report' do
+
+    before do
+      login
+    end
+
+    context 'when displaying json' do
+      it 'properly displays the echo json report' do
+        visit loss_report_collections_path(echo_id, format:'json')
```

Comment:
As I mentioned in the helper/fixture file, you it is better if we don't use a hard coded concept id. I think for this you might want to do a search for this collection based on short name or something, and then go to the show page. Because there is no button yet, you might want to either grab the concept id from the url (I can provide an example if you need) or grab the url to construct the path you need. This may mean that it makes more sense to restructure this file a bit (i.e., grouped by collection native format vs output format).